core/src/main/java/org/everit/json/schema/SchemaException.java

```diff
@@ -1,9 +1,11 @@
 package org.everit.json.schema;
 
+import java8.util.stream.Collectors;
+import java8.util.stream.StreamSupport;
+
 import static java.lang.String.format;
 import static java.util.Arrays.asList;
-import static java.util.Objects.requireNonNull;
-import static java.util.stream.Collectors.joining;
```

Comment:
Why don't you use static import for java8 library here as well like "import static java8.util.stream.Collectors.joining;" instead of importing the whole class?